src/main/resources/io/jenkins/plugins/designlibrary/Dialogs/dialogs.css

```diff
@@ -0,0 +1,3 @@
+#toggle-switch-formsubmit {
```

Comment:
Would you mind having this as a `.scss` file inside the `pages` folder - thanks :)

Reply:
> Would you mind having this as a `.scss` file inside the `pages` folder - thanks :)

done